crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -36,9 +39,9 @@ pub enum TupleParentheses {
     /// for a, b in x:
     ///     pass
     /// ```
-    /// Black still does use parentheses in this position if the group breaks or magic trailing
+    /// Black still does use parentheses in these positions if the group breaks or magic trailing
     /// comma is used.
-    StripInsideForLoop,
+    NeverPreserve,
```

Comment:
Similarly, could you add these examples?

```python
for (a,) in []:
    pass
for a, b in []:
    pass
for a, b in []:  # Strips parentheses
    pass
for (
    aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
    b,
) in []:
    pass
```

Reply:
This one makes me think you want fixtures, not comment examples. I'll add the comment examples for the last suggestion. Do you mean to add these as fixtures? This is what's there currently:

```rs
    /// Handle the special cases where we don't include parentheses if they are not required.
    ///
    /// Normally, black keeps parentheses, but in the case of for loops it formats
    /// ```python
    /// for (a, b) in x:
    ///     pass
    /// ```
    /// to
    /// ```python
    /// for a, b in x:
    ///     pass
    /// ```
    /// Black still does use parentheses in these positions if the group breaks or magic trailing
    /// comma is used.
    NeverPreserve,
```

Reply:
395404b5e9ef2769bc9f0c32473a3f083abd5933 (potentially resolved)